BookStore/resources/views/books/cart-page.blade.php

```diff
@@ -1,37 +1,66 @@
 @extends('layouts.layouts')
 @section('title', 'Cart')
 @section('content')
+    @php
+    $id = 1;
+    $index = 0;
```

Comment:
add index in foreach like this
`@foreach (session(cartItems) as $index => $cartItem)`